lib/Doctrine/ORM/Cache/QueryCacheEntry.php

```diff
@@ -38,18 +38,18 @@ class QueryCacheEntry implements CacheEntry
     /**
      * READ-ONLY: Public only for performance reasons, it should be considered immutable.
      *
-     * @var integer Time creation of this cache entry
+     * @var float Time creation of this cache entry
```

Comment:
I did but I definitely forgot to change `($entry->time + $key->lifetime) > time()` to `((int) $entry->time + $key->lifetime) > time()` on the validator